css/styles.css

```diff
@@ -50,6 +69,43 @@ body {
   margin: 0 10px 0 0;
 }
 
+.work-content a {

```

Comment:
Please remove the border to stick to Figma

<img width="654" alt="image" src="https://user-images.githubusercontent.com/7243132/170840357-7106bccd-ee12-425e-a2a3-b1d8c80c9a03.png">